test/cpp_extensions/cpp_c10d_extension.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <torch/extension.h>
+
+#include <deque>
+#include <exception>
+#include <memory>
+#include <mutex>
+#include <thread>
+#include <vector>
+
+#include <c10d/ProcessGroup.hpp>
+#include <c10d/Types.hpp>
+#include <c10d/Utils.hpp>
+#include <c10d/Store.hpp>
+
+namespace c10d {
+
+//
+// ProcessGroupC10dTest implements a dummy bindings for c10d.
+//
+
+class ProcessGroupC10dTest : public ProcessGroup {
+ public:
+
+  explicit ProcessGroupC10dTest(int rank = -1, int size = -1);
+  virtual ~ProcessGroupC10dTest();
+
+  std::shared_ptr<ProcessGroup::Work> broadcast(
```

Comment:
Just style-related comment - I would prefer to have the same ordering for functions in header and cpp files just to be aligned.

Reply:
sovled.